README.md

```diff
@@ -81,10 +81,18 @@ Protobuf provides the `protoc` executable to encode, decode and **generate** lan
 
 #### 1. Install the protoc plugin
 
+##### macOS:
 ```
 brew install jeromegn/tap/protoc-gen-crystal
 ```
 
+##### Ubuntu:
+
+```
+crystal build bin/protoc-gen-crystal.cr
+cp protoc-gen-crystal ~/bin/
```

Comment:
Is `~/bin/` in the `$PATH` by default on Ubuntu? If not it'd be best to specify that it just needs to be in folder included in `$PATH`.